packages/augur-core/source/contracts/reporting/ShareToken.sol

```diff
@@ -8,13 +8,15 @@ import 'ROOT/libraries/Initializable.sol';
 import 'ROOT/reporting/IMarket.sol';
 import 'ROOT/IAugur.sol';
 import 'ROOT/CashSender.sol';
+import 'ROOT/libraries/TokenId.sol';
 
 
 /**
  * @title Share Token
  * @notice ERC1155 contract to hold all Augur share token balances
  */
 contract ShareToken is ITyped, Initializable, ERC1155, IShareToken, ReentrancyGuard, CashSender {
+    using TokenId for IShareToken;
```

Comment:
I believe this line can be deleted.  Not only are you not using it anywhere, I don't believe it is actually valid since `TokenId` is not a library that operates on `IShareToken`.